src/main/java/carsharingservice/controller/AuthenticationController.java

```diff
@@ -0,0 +1,41 @@
+package carsharingservice.controller;
+
+import carsharingservice.dto.auth.login.UserLoginRequestDto;
+import carsharingservice.dto.auth.login.UserLoginResponseDto;
+import carsharingservice.dto.auth.registration.UserRegistrationRequestDto;
+import carsharingservice.dto.user.UserResponseDto;
+import carsharingservice.exception.RegistrationException;
+import carsharingservice.security.AuthenticationService;
+import carsharingservice.service.user.UserService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/auth")
```

Comment:
```suggestion
@RequestMapping("/auth")
```

add `server.servlet.context-path=/api` to application.properties